src/schema/index.js

```diff
@@ -59,6 +59,7 @@ type Transaction {
   type: _TransactionType!
   status: _TransactionStatus!
   txid: String
+  createdBlock: Int
```

Comment:
what's the difference between `createdBlock` & `blockNum`?

Reply:
`createdBlock` is for the block when the tx is created. we only track `createdTime` right now but i needed this for `transfer` qtum tx. 

they changed something in qtum client where it doesnt return the block num anymore in `gettransaction` which is how i validate transfer qtum txs. need to know the createdBlock to be able to compare it for confirmation purposes.

Reply:
@dwalintukan i c. Need find a new rpc call then. Can we add a `TODO` on this line and I track it #199